test/CMakeLists.txt

```diff
@@ -205,6 +205,12 @@ if(NRN_ENABLE_PYTHON AND PYTEST_FOUND)
     ENVIRONMENT COVERAGE_FILE=.coverage.cover_tests
     COMMAND ${PYTHON_EXECUTABLE} ${pytest} ./test/cover
     SCRIPT_PATTERNS test/cover/*.py test/cover/*.json)
+  nrn_add_test_group(NAME example_nmodl MODFILE_PATTERNS share/examples/nrniv/nmodl/*.mod
+                                                         share/examples/nrniv/nmodl/*.inc)
+  nrn_add_test(
+    GROUP example_nmodl
+    NAME example_nmodl
+    COMMAND special -c [=[ quit() ]=])
```

Comment:
Is it possible to execute some of the `.hoc` scripts in the directory to pick up some more coverage?

Reply:
Have not been able to develop a COMMAND that does the above loop.

Reply:
Done in f8659a3